### ISSUE TITLE
Error: The "build" command is only allowed within an existing package.

### ISSUE THREAD

**Issue by idgserpro** [plone/plonecli#59]
* Plone CLI version: 1.1
* Python version: 2.7.17
* Operating System: Ubuntu 18.04

### Description

When I try to run `plonecli build` in the package:

https://github.com/plonegovbr/brasil.gov.agenda

I get the error:

`Error: The "build" command is only allowed within an existing package.`

### What I Did

```
cd /tmp
git clone https://github.com/plonegovbr/brasil.gov.agenda
cd brasil.gov.agenda
plonecli build
Usage: plonecli build [OPTIONS]

Error: The "build" command is only allowed within an existing package.
```
Why is this package not considered a package by plonecli?

**Comment by idgserpro** [comment]
Adding a `bobtemplate.cfg` solves the problem:

```
[main]
version = 4.3.x
template = buildout
```

Can we have a better error message? Stating this file is needed, if possible with an example?

**Comment by idgserpro** [comment]
By the way. Is this `bobtemplate.cfg` file really necessary for the `plonecli build` command?

**Comment by MrTango** [comment]
this is fixed in the new version 2.0b1, the general handling of outocompletion and command filtering is better now.
